prime-router/src/main/kotlin/cli/tests/WatersAuthTests.kt

```diff
@@ -708,64 +709,108 @@ class Server2ServerAuthTests : CoolTest() {
         if (environment == Environment.PROD) error("Can't create simple_report test data in PROD")
         val org1 = "simple_report"
         var sender1 = createNewSenderForExistingOrg(mySenderName, org1)
-        // Right now in order to access submission history you must have the default scope
-        // This attaches the ignore.default.report scope claim to the ignore.temporary_sender_auth_test.report Sender.
-        val scope1 = "$org1.default.report"
+        // Test various functionality using the general <orgname>.*.user role.
+        val scope1 = "$org1.*.user"
+        // Scope <org>.<sender>.report only gives access to submit to that org only.  Doesn't work for history GETs.
         val uploadReportScope1 = "$org1.$mySenderName.report"
         // Submit this new scope and public key to the Settings store, associated with this Sender.
         sender1 = saveServer2ServerKey(sender1, end2EndExampleRSAPublicKeyStr, kid, scope1)
         sender1 = saveServer2ServerKey(sender1, end2EndExampleRSAPublicKeyStr, kid, uploadReportScope1)
 
         val org2 = "ignore"
         var sender2 = createNewSenderForExistingOrg(mySenderName, org2)
-        // Right now in order to access submission history you must have the default scope
-        // This attaches the ignore.default.report scope claim to the ignore.temporary_sender_auth_test.report Sender.
-        val scope2 = "$org2.BOGOSITY.report"
-        val uploadReportScope2 = "$org2.$mySenderName.report"
+        // Test various functionality using the general <orgname>.*.admin role.
+        val scope2 = "$org2.*.admin"
         // Submit this new scope and public key to the Settings store, associated with this Sender.
         sender2 = saveServer2ServerKey(sender2, end2EndExampleRSAPublicKeyStr, kid, scope2)
-        sender2 = saveServer2ServerKey(sender2, end2EndExampleRSAPublicKeyStr, kid, uploadReportScope2)
 
         try {
             val myFakeReportFile = OktaAuthTests.createFakeReport(sender1)
 
-            // Now request 5-minute tokens for the first org
```

Comment:
`uploadReportScope1` is a nice example of a very narrowly defined scope - the ability to upload to this specific receiver with this specific org.